esi_ui/static/dashboard/esi/lessee/offers/esi-offers-table.controller.js

```diff
@@ -69,6 +72,20 @@
         });
       });
     }
+
+    function getOfferAvailabilityStatusColor(status) {
+      return status === 'available' ? 'green' : 'red';
+    }
+  
+    function getOfferAvailabilityColor(availabilityStart) {
+      const currentTime = new Date(); 
+      const availabilityDateTime = new Date(availabilityStart.replace(" ", "T") + "Z");
+      
+      const timeDifference = (availabilityDateTime - currentTime) / 60000;
+  
+      return timeDifference <= 0 ? 'green' : 'red';
```

Comment:
Can we use brown instead of red? The offer is probably still available, just not at the present moment.

Reply:
@tzumainn I did brown for now but to me give similar vibe as red let me know if you need blue after you see it.